### PR TITLE
Add gogs support squashed

### PR DESCRIPTION
**Pull request by pepper-jk**
This introduces support for Gogs export/import using the Gogs API. The Gogs API however does not offer a full GitHub API feature set yet. Although its existing interfaces are the same.

Supported:
- Issues
- Labels

Not supported:
- Pull Requests

I imported [this repo](https://github.com/tklab-tud/ID2T) to GitHub from [here](https://git.tk.informatik.tu-darmstadt.de/SPIN/ID2T-toolkit). Feel free to check it out.

If you want to test it yourself, just use [try Gogs](https://try.gogs.io/).

p.s. On my last pull request you removed the signature from my commit. I'm assuming this was not on purpose, but I would appreciate it if you could avoid this in the future.

### CODE REVIEW

**Comment by pepper-jk** [comment]
Any updates?

**Comment by pepper-jk** [comment]
updates?

File: gh-issues-import-ng.py [hunk 1]
```diff
@@ -29,14 +29,15 @@ class state:
...
 http_error_messages = {}
-http_error_messages[401] = "ERROR: There was a problem during authentication.\nDouble check that your username and " \
-                           "password are correct, and that you have permission to read from or write to the specified" \
-                           " repositories."
-http_error_messages[403] = http_error_messages[401]     # Basically the same problem.
-                                                        # GitHub returns 403 instead to prevent abuse.
-http_error_messages[404] = "ERROR: Unable to find the specified repository.\nDouble check the spelling for the source" \
-                           " and target repositories. If either repository is private, make sure the specified user " \
-                           "is allowed access to it."
+http_error_messages[401] = "HTTP ERROR 401: There was a problem during authentication.\n" \
```
Comment (leemgs) [inline gh-issues-import-ng.py:32]: Looks good.

File: config.ini.sample [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Rename this file to config.ini
 
 [source]
+#for gogs support use:
```
Comment (leemgs) [inline config.ini.sample:4]: I like this proposal to handle gogos as well as GitHub.
👍

**Review by leemgs — APPROVED** [review]
LGTM

**Pull request merged**